### DISCUSSION TITLE
Should we worry about imports perfomance in handlers?

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#27] [Q&A]
My dialogue with chatgpt about it: https://chat.openai.com/share/1e0f86ff-2094-441c-994a-d797076edb22

**Comment by konard** [comment]
https://chat.openai.com/share/1e0f86ff-2094-441c-994a-d797076edb22 link is broken.
Anyway, there are always multiple ways to make optimizations later.

**Reply by FreePhoenix888** [reply]
🤔 
![image](https://github.com/deep-foundation/Discussions/assets/66206278/49620674-9dd3-4152-8d23-40c58fd0067e)
I can open it

**Reply by FreePhoenix888** [reply]
Even in incognito window
![image](https://github.com/deep-foundation/Discussions/assets/66206278/c7473f5f-5dbc-4838-87d4-2722a1641be4)
